File: src/main/java/jkmau5/alternativeenergy/AltEngCreativeTab.scala
package jkmau5.alternativeenergy

import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.ItemStack
import jkmau5.alternativeenergy.world.blocks.AltEngBlocks

/**
 * No description given
 *
 * @author jk-5
 */
object AltEngCreativeTab extends CreativeTabs("altEng") {
  def init() = {}
  override def getIconItemStack = new ItemStack(AltEngBlocks.blockPowerCable, 1, 0)
}

File: src/main/java/jkmau5/alternativeenergy/AltEngCreativeTab.java
package jkmau5.alternativeenergy;

import jkmau5.alternativeenergy.world.item.AltEngItems;
import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;

/**
 * No description given
 *
 * @author jk-5
 */
public class AltEngCreativeTab extends CreativeTabs {

    public AltEngCreativeTab() {
        super("altEng");
    }

    @Override
    public ItemStack getIconItemStack() {
        return new ItemStack(AltEngItems.itemUpgrade, 1, 0);
    }
}
